TPL/flann-1.8.4/src/cpp/.gitignore

```diff
@@ -0,0 +1,3 @@
+CMakeFiles
```

Comment:
This file should be unnecessary due to the ``.gitignore`` file two directories up?